build/build-gh-pages.js

```diff
@@ -86,9 +112,27 @@ function zipFileDownloadLink(linkText) {
   `;
 }
 
+
 function iconCards(svgFileNames) {
-  return `
-    <div class="icon-cards">${svgFileNames.map(iconCard).join("")}</div>
+  let warningGroups = [];
+  warningTypes.map((warning) => {
```

Comment:
You can do `warningGroups = warningTypes.map((warning) => {` and return an object in the map function.